### DISCUSSION TITLE
PR target

### DISCUSSION THREAD

**Discussion by violette** [renovatebot/renovate#10936] [Request Help] [answered]
Hi,

Thank you for your work. I still have an issue with the configuration if you can help me.
We are based on a stash server. I have a fork of the main repo, each dev has one.
I would like to launch the bot (self-hosted using the docker image, version 25.56.7) on my fork, without adding any configuration file to the main repo(origin master).

My configuration is:

```
{
	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
	"extends": [
		"config:base"
	],
	"endpoint": "https://my.stash.repo/",
	"repositories": [
		"~username/myProjectName"
	],
	"platform": "bitbucket-server",
    ...
	"automerge": false,
	"autodiscover": false,
	"includeForks": true,
	"onboarding": false,
	"requireConfig": false
}
```

Actually, when I launch that, it creates branches and PR on my fork, meaning: **source** my fork to **destination** my fork master.
but I'd like to create the PRs on the original master repo instead. (**source** my fork to **destination** original master repo).

It that possible?

Thanks,

**Comment by rarkins** [comment, marked as answer]
Not currently possible, would need some development work to achieve. On GitHub we have a concept called forkMode which is similar.

**Reply by violette** [reply]
Oh :( Is that something planned to be implemented?

**Reply by rarkins** [reply]
Not currently planned. No feature request has been submitted and Stash usage has been decreasing in general so it would require a user like yourself interested in implementing it.

**Reply by violette** [reply]
Ok, I'll open a feature request.
In the meanwhile, could you just point me the file managing the stash PR creation? I'll have a look as soon as possible.
